bindings/matrix-sdk-ffi/src/sliding_sync.rs

```diff
@@ -199,6 +199,12 @@ impl SlidingSyncRoom {
             .map(|room| Arc::new(Room::with_timeline(room, self.timeline.clone())))
     }
 
+    pub fn avatar_url(&self) -> Option<String> {
+        self.client
+            .get_room(self.inner.room_id())
+            .and_then(|room| room.avatar_url().map(|url| url.into()))
```

Comment:
I think this would be more readable with `Some()` + `?` instead of `and_then` + `map`, as in my previous suggestion.

Reply:
Ah, dang it, you're right. On it